spinnaker/spinnaker_system/google_smoke_test.py

```diff
@@ -51,12 +51,14 @@
 import citest.service_testing as st
 
 # Spinnaker modules.
+from google_http_lb_upsert_scenario import GoogleHttpLoadBalancerTestScenario
+
 import spinnaker_testing as sk
 import spinnaker_testing.gate as gate
 import citest.base
 
 
-class GoogleSmokeTestScenario(sk.SpinnakerTestScenario):
+class GoogleSmokeTestScenario(GoogleHttpLoadBalancerTestScenario):
```

Comment:
I would rather you didnt do this inheritance.
Could you do something like GoogleHttpLoadBalancerTestScenario(self.scenario.bindings, self.scenario.agent).upsert_min_load_balancer())